tests/baselines/reference/user/adonis-framework.log

```diff
@@ -1,6 +1,9 @@
 Exit Code: 1
 Standard output:
 node_modules/adonis-framework/lib/util.js(24,13): error TS2304: Cannot find name 'Mixed'.
+node_modules/adonis-framework/lib/util.js(29,6): error TS2339: Property 'existy' does not exist on type '{}'.
```

Comment:
The line `const util = exports = module.exports = {}` now results in `util: {}` not `util: { [x: string]: any}`. I'll open a bug.

Reply:
Opened #25687 to track this error.